Containers/mastercontainer/Dockerfile

```diff
@@ -7,22 +7,24 @@ FROM caddy:2.6.4-alpine as caddy
 # From https://github.com/docker-library/php/blob/master/8.1/alpine3.17/fpm/Dockerfile
 FROM php:8.1.18-fpm-alpine3.17
 
-RUN set -ex; \
-    apk add --no-cache shadow; \
-    groupmod -g 333 xfs; \
-    usermod -u 333 -g 333 xfs; \
-    groupmod -g 33 www-data; \
-    usermod -u 33 -g 33 www-data
-
 EXPOSE 80
 EXPOSE 8080
 EXPOSE 8443
 
-RUN mkdir -p /var/www/docker-aio;
+COPY mastercontainer.conf /etc/apache2/sites-available/mastercontainer.conf
```

Comment:
why is it up here?

Reply:
because it was in the middle of RUN steps, I will put it down

Reply:
done